src/components/Dissolution/CompleteResolutionSummary.vue

```diff
@@ -94,9 +94,9 @@
             rows="1"
             auto-grow
             readonly
-            filled
+            variant="filled"
             hide-details
-            background-color="white"
+            bg-color="white"
```

Comment:
Why would they change this? I see no reason and it's going to cost a lot of companies $$.

Reply:
Totally Agree Sev. And it's such a small change too.
![CompleteResolutionSummary](https://github.com/bcgov/business-create-ui/assets/122301442/90407de1-4e99-41e0-a0dd-6f4094ea3404)

Reply:
Hey, eslint is really helpful!